index.js

```diff
@@ -24,57 +20,65 @@ function encrypt (data, key) {
 /* website will generate pub key, send to getKey
    which will return a JWT for use in sending request */
 export const getRequestURL = IOpipe((event, context, callback) => {
+  context.iopipe.mark.start('jwt-sign');
   jsonwebtoken.sign(
-      { data: 'hello world' },
-      keys.private,
-      { algorithm: 'RS256' },
-      (err, token) => {
-    if (err) {
-      context.iopipe.log("error", err);
-      context.iopipe.log("token", token);
-      callback(null, { "statusCode": "400", "body": err });
-      return;
-    }
+    { data: 'hello world' },
+    keys.private,
+    { algorithm: 'RS256' },
+    (err, token) => {
+      if (err) {
+        context.iopipe.log("error", err);
+        context.iopipe.log("token", token);
+        callback(null, { "statusCode": "400", "body": err });
+        return;
+      }
 
-    var hash = crypto.createHash('sha256');
-    hash.update(token);
-    var fileName = `${hash.digest('hex')}.json`;
+      var hash = crypto.createHash('sha256');
+      hash.update(token);
+      var fileName = `${hash.digest('hex')}.json`;
 
-    callback(null, {
-      "statusCode": 200,
-      "headers": {
-        "Access-Control-Allow-Origin" : "*", // Required for CORS support to work
-        "Access-Control-Allow-Credentials" : true // Required for cookies, authorization headers with HTTPS
-      },
-      "body": JSON.stringify({
-        "requestTo": `${event.headers['X-Forwarded-Proto']}://${event.headers['Host']}/req/${token}\n`,
-        "requestFrom": `https://response.lol/${fileName}`,
-        "requestFromToken": token,
-        "requestFromCurl": `curl -H 'Authorization: Bearer ${token}' https://response.lol/${fileName}`
-      }, null, 2)
-    });
-  });
+      context.iopipe.mark.end('jwt-sign');
+      callback(null, {
+        "statusCode": 200,
+        "headers": {
+          "Access-Control-Allow-Origin" : "*", // Required for CORS support to work
+          "Access-Control-Allow-Credentials" : true // Required for cookies, authorization headers with HTTPS
+        },
+        "body": JSON.stringify({
+          "requestTo": `${event.headers['X-Forwarded-Proto']}://${event.headers['Host']}/req/${token}\n`,
+          "requestFrom": `${event.headers['X-Forwarded-Proto']}://${fileName}`,
+          "requestFromToken": token,
+          "requestFromCurl": `curl -H 'Authorization: Bearer ${token}' ${event.headers['X-Forwarded-Proto']}://${fileName}`
+        }, null, 2)
+      });
+    }
+  );
 });
 
 // eslint-disable-next-line import/prefer-default-export
 export const handleRequest = IOpipe((event, context, callback) => {
-  console.log(event);
   var pathJwt = event.pathParameters.token;
   jsonwebtoken.verify(pathJwt, keys.public,
     { algorithms: [ 'RS256' ] },
     (err, decodedJwt) => {
-      if (err) return callback(null, {
-        "statusCode": 400,
-        "body": `Error in pathJwt: ${err}\n`
-      });
+      if (err) {
+        context.iopipe.metric('jwt-error', true);
```

Comment:
either log or metric! Metric is the new name for log

Reply:
(here you're making two custom metrics of the same name)

Reply:
Got it will fix, thanks!